core/src/main/scripts/importer/validateData.py

```diff
@@ -2941,7 +2941,6 @@ def checkLine(self, data):
                     self.logger.error(
                             'Value in DFS_STATUS column is not 0:DiseaseFree, '
                             '1:Recurred/Progressed, 1:Recurred, 1:Progressed',
-                            'DiseaseFree, Recurred/Progressed, Recurred or Progressed',
```

Comment:
@pvannierop I see, I was wondering why we delete `DiseaseFree, Recurred/Progressed, Recurred or Progressed`. Now I get it. You are right. We don't support formats like these `DiseaseFree, Recurred/Progressed, Recurred or Progressed` anymore, we only support values with the prefix `0` or `1`. So we should delete this line.